### PR TITLE
Improve documentation of both rotational and translational sensors

### PR DESCRIPTION
**Pull request by tobolar**
Esp. improve format of variables and Modelica code according to HTML documentation conventions.

### CODE REVIEW

**Review by MartinOtter — APPROVED** [review]

File: Modelica/Mechanics/Translational/Sensors/PowerSensor.mo [hunk 1]
```diff
@@ -1,6 +1,6 @@
 within Modelica.Mechanics.Translational.Sensors;
 model PowerSensor
-  "Ideal sensor to measure the power between two flanges (= flange_a.f*der(flange_a.s))"
+  "Ideal sensor to measure the power between two flanges (= flange_a.f * der(flange_a.s))"
```
Comment (christiankral) [inline Modelica/Mechanics/Translational/Sensors/PowerSensor.mo:3]:
```suggestion
  "Ideal sensor to measure the power between two flanges"
```

File: Modelica/Mechanics/Translational/Sensors/PowerSensor.mo [hunk 2]
```diff
@@ -14,8 +14,8 @@ equation
   annotation (
     Documentation(info="<html>
 <p>
-Measures the <strong>power between two flanges</strong> in an ideal way
-and provides the result as output signal <strong>power</strong>
+Measures the <em>power</em>&nbsp;<var>P</var> between two flanges
```
Comment (christiankral) [inline Modelica/Mechanics/Translational/Sensors/PowerSensor.mo:17]:
```suggestion
Measures <code>power  = flange_a.f * der(flange_a.s)</code> between two flanges
```

**Review by christiankral — CHANGES_REQUESTED** [review]
I suggest to move the calculation equation of the power sensors to the HTML documentation

File: Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo [hunk 3]
```diff
@@ -1,6 +1,6 @@
 within Modelica.Mechanics.Rotational.Sensors;
 model PowerSensor
-  "Ideal sensor to measure the power between two flanges (= flange_a.tau*der(flange_a.phi))"
+  "Ideal sensor to measure the power between two flanges (= flange_a.tau * der(flange_a.phi))"
```
[the remark this answers is not in the document]
Comment (tobolar) [reply Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo]: Resolved in d71ceb225c03046a6f897f0640d52b935fd018e5

File: Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo [hunk 4]
```diff
@@ -16,8 +16,8 @@ equation
   annotation (
     Documentation(info="<html>
 <p>
-Measures the <strong>power between two flanges</strong> in an ideal way
-and provides the result as output signal <strong>power</strong>
+Measures the <em>power</em>&nbsp;<var>P</var> between two flanges
```
[the remark this answers is not in the document]
Comment (tobolar) [reply Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo:19]: > As the variableP is not used at all it makes no sense to refer to it.

This also concerns other sensors where a variable is always written. But I think we remove it there as well.

[the remark this answers is not in the document]
Comment (tobolar) [reply Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo]: Done in https://teamsites-extranet.dlr.de/ts/PT-Board-Strassenverkehr/GFPlanung%202022ff/3%20VuV4NGC%20(Methoden,%20Prozesse%20und%20Werkzeugketten%20f%C3%BCr%20die%20Validierung%20und%20Verifikation%20von%20NGC)/TP%207000/VorlageAnhangProjektplan_TP7000.docx?web=1

File: Modelica/Mechanics/Translational/Sensors/PowerSensor.mo (hunk 1 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational/Sensors/PowerSensor.mo]: Changed, see above.

File: Modelica/Mechanics/Translational/Sensors/PowerSensor.mo (hunk 2 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational/Sensors/PowerSensor.mo]: Changed, see above.

File: Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo (hunk 4 above)
[the remark this answers is not in the document]
Comment (tobolar) [reply Modelica/Mechanics/Rotational/Sensors/PowerSensor.mo]: Done in 33b2ffad0d01ba0d012eded10b34f8686bbfe34d

**Review by hubertus65 — APPROVED** [review]
Looks all good.

**Review by hubertus65 — APPROVED** [review]

**Review by hubertus65 — APPROVED** [review]

**Review by hubertus65 — APPROVED** [review]

**Pull request merged**